sig/coprocessor/membership.md

```diff
@@ -0,0 +1,22 @@
+# Coprocessor SIG Members
+
+## Maintainers
+
+- [breeswish](https://github.com/breeswish)
```

Comment:
These two are Tech Leader instead of Maintainer?

Reply:
I think the `Maintainer` is more appropriate, see: https://github.com/pingcap/tidb-lightning/blob/master/MAINTAINERS.md